03-csv-vector-raster-plotting.Rmd

```diff
@@ -472,24 +488,21 @@ xmax <- originalPlotExtent@xmax
 ymin <- originalPlotExtent@ymin
 ymax <- newPlotExtent@ymax
 
-
-```
-
-```{r challenge-plot, echo=FALSE}
-
+#3 again... re-plot
 #try again but this time specify the x and ylims
 #note: we could also write a function that would harvest the smallest and largest
 #x and y values from an extent object. This is beyond the scope of this lesson.
 plot(plot.locationsSp_HARV, 
      main="NEON Harvard Forest Field Site\nVegetation & Phenology Plots",
      pch=8,
+     col="purple",
      xlim=c(xmin,xmax),
      ylim=c(ymin,ymax))
 
 plot(newPlot.Sp.HARV.UTM, 
      add=TRUE, pch=20, col="darkgreen")
 
-#to create a custom legend, we need to fake it
+#to create a custom legend, we need specify the text as it isn't in the data
```

Comment:
In the `newPlots...` object, subtype is specified as phenology.  In the `plotLocations...` object there is no subtype attribute. 

Changed to: "#to create a custom legend, we need specify the text for the type of each plot".  
Let me know if we need to do anything to the files or you want me to change this text.